modules/jaeger-deployment-best-practices.adoc

```diff
@@ -2,11 +2,9 @@
 This module included in the following assemblies:
 *  /jaeger/jaeger_install/rhbjaeger-deploying.adoc
 ////
-
+:pantheon-module-type: CONCEPT
```

Comment:
```suggestion
:_content-type: CONCEPT
```
Global comment for similar occurances.

**Question:** 

- Any particular reason why this is different from what is suggested here: https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=JUP&title=Jupiter+Source+files+readiness+Checklist#JupiterSourcefilesreadinessChecklist-Ifyouarenotadheringtonamingconventions ?

Reply:
As far as I know, the current standard is `:_content-type: CONCEPT`
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#content-type-attributes